code/modules/hydroponics/hydroponics.dm

```diff
@@ -440,16 +440,21 @@
 		dead = 1
 
 
-/obj/machinery/hydroponics/proc/mutatepest()
+
+/obj/machinery/hydroponics/proc/mutatepest(mob/user)
 	if(pestlevel > 5)
+		message_admins("[ADMIN_LOKUPFLW] caused spiderling pests to spawn in a hydro tray")
+		log_game("[key_name(user)] caused spiderling pests to spawn in a hydro tray")
 		visible_message("<span class='warning'>The pests seem to behave oddly...</span>")
-		for(var/i=0, i<3, i++)
-			var/obj/structure/spider/spiderling/S = new(src.loc)
-			S.grow_as = /mob/living/simple_animal/hostile/poison/giant_spider/hunter
+		spawn_atom(/obj/structure/spider/spiderling, src, 3, src, "post_spiderling_spawn")

```

Comment:
admin_spawn arg being passed src? I think you need to either name your arguments or give it FALSE for admin_spawn